src/components/DialogInStepperWrapper/DialogInStepperWrapper.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react'
+
+import { DialogModalContext, StepperContext } from '../Contexts'
+
+const DialogInStepperWrapper = () => {
+  const { currentPages } = useContext(DialogModalContext)
+  const { currentStep } = useContext(StepperContext)
+
+  return currentPages.map((page, i) => (
+    <React.Fragment key={i}>
+      {page.pageNumber === currentStep && (
+        <>
+          {/* <DialogInStepper ... title={via DialogModalContext} content={<LazyLoad currentPage={page} />} ... */}
```

Comment:
Is that commented line wanted?

Reply:
Oui c'est pour donner une idée de ce qui va arriver ensuite, un peu plus de contexte :)